src/components/results/sensitivity-analysis/sensitivity-analysis-result-tab.js

```diff
@@ -23,35 +23,40 @@ const SensitivityAnalysisResultTab = ({ studyUuid, nodeUuid }) => {
     const [sensiKindIndex, setSensiKindIndex] = useState(0);
     const [page, setPage] = useState(0);
 
-    const { updateFilter, filterSelector, initFilters, rowFilters } =
-        useRowFilter(DATA_KEY_TO_FILTER_KEY);
+    const { updateFilter, filterSelector, initFilters } = useRowFilter(
+        DATA_KEY_TO_FILTER_KEY
+    );
 
     // Add default sort on sensitivity col
-    const defaultSortColumn = DATA_KEY_TO_SORT_KEY.value;
+    const defaultSortColumn =
+        DATA_KEY_TO_SORT_KEY[nOrNkIndex ? 'valueAfter' : 'value'];
     const defaultSortOrder = SORT_WAYS.desc;
     const { onSortChanged, sortConfig, initSort } = useAgGridSort(
         DATA_KEY_TO_SORT_KEY,
-        { colKey: defaultSortColumn, sortWay: defaultSortOrder }
+        {
+            colKey: defaultSortColumn,
+            sortWay: defaultSortOrder,
+        }
     );
 
-    const initTable = () => {
+    const initTable = (newNOrNKIndex) => {
```

Comment:
```suggestion
    const initTable = (nOrNkIndex) => {
```
It's not always new, cf line 53

Reply:
done, thank you